src/nativegasestimator.rs

```diff
@@ -41,8 +41,6 @@ pub struct Params {
     pub extra_priority_fee_boost: f64,
     // priority fee offered when there are no recent transactions
     pub fallback_priority_fee: f64,
-    // a coefficient to multiply max_fee_per_gas with, in order to increase chances of transaction inclusion
-    pub bump_cap_coefficient: f64,
```

Comment:
I would be in favor of updating the parameter name and comment instead of introducing 2 new magic numbers.
Or maybe keep it as a named constant somewhere if it doesn't make sense to keep the value configurable.

Reply:
Indeed. fixed.